server/routes/barcode/contacts/updateContactValidator.test.ts

```diff
@@ -76,7 +76,7 @@ describe('updateContactValidator', () => {
     mockValidatePrisonId.mockReturnValue([])
     mockValidatePrisonerDob.mockReturnValue(['some-dob-error'])
 
-    expect(validateContact(form)).toEqual([{ href: '#dob', text: 'some-dob-error' }])
+    expect(validateContact(form)).toEqual([{ href: '#dob-day', text: 'some-dob-error' }])
```

Comment:
We chose not to differentiate date errors by day/month/year but this meant that the link in the error header didn't point at an input field. The broken link confused the tester using a screen reader.

Reply:
There was a separate low priority issue found that suggests we should split out errors by day/month/year which might also mitigate this issue - but I'm still trying to understand how that split could work without getting very complicated.